rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMServiceAsync.java

```diff
@@ -146,6 +146,25 @@ void editNodeSource(String sessionId, String nodeSourceName, String infrastructu
             String[] policyParameters, String[] policyFileParameters, String nodesRecoverable,
             AsyncCallback<String> callback) throws RestServerException, ServiceException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)